src/Tribe/RSVP.php

```diff
@@ -1207,7 +1212,27 @@ public function front_end_tickets_form( $content ) {
 	 */
 	public function login_required() {
 		$requirements = (array) tribe_get_option( 'ticket-authentication-requirements', array() );
-		return in_array( 'event-tickets_rsvp', $requirements );
+		return in_array( 'event-tickets_rsvp', $requirements, true );
+	}
+
+	/**
+	 * Determine whether to use new RSVP views.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool Whether to use new RSVP views.
+	 */
+	public function use_new_views() {
+		// Determine if ET was installed at version 4.12.2+.
+		$should_default_to_on = ! tribe_installed_before( 'Tribe__Tickets__Main', '4.12.2' );
+
+		return (
+			(
+				defined( 'TRIBE_TICKETS_RSVP_NEW_VIEWS' )
+				&& TRIBE_TICKETS_RSVP_NEW_VIEWS
+			)
+			|| (boolean) tribe_get_option( 'rsvp_use_new_views', $should_default_to_on )
```

Comment:
Maybe we can prefix the option? Not sure how we do it normally, but I feel we should prefix it.

Reply:
It's inside of our own Option so it would be safe I think.